### DISCUSSION TITLE
[Feat][Doc] Are there plans to develop Chinese documentation?

### DISCUSSION THREAD

**Discussion by songjianet** [vuejs/vuefire#1246] [Ideas]
### What problem is this solving

There are many developers in mainland China using it, so it would be more friendly to have a better Chinese documentation.

### Proposed solution

I am willing to help contribute Chinese documentation.

**Comment by posva** [comment]
It's good to know! I don't have plans on writing the documentation in any other language than English (because it's a lot of work already 😄) but feel free to create a fork with the translation and I will be able to link it.

However, note that the current docs are still a very early WIP, so it's not a good idea to start translating them. I would wait at least a month or until a stable version of VueFire v3 is release. Note the roadmap is at #1241

**Reply by posva** [reply]
It's now safe to fork the docs if anybody wants to contribute!
